packages/tracing-internal/src/browser/metrics/utils.ts

```diff
@@ -12,9 +12,14 @@ export function isMeasurementValue(value: unknown): value is number {
  * Helper function to start child on transactions. This function will make sure that the transaction will
  * use the start timestamp of the created child span if it is earlier than the transactions actual
  * start timestamp.
+ *
+ * Note: this will not be possible anymore in v8,
+ * unless we do some special handling for browser here...
  */
 export function _startChild(transaction: Transaction, { startTimestamp, ...ctx }: SpanContext): Span {
```

Comment:
This is really the main place I found where we are overwriting the start time in a way that is not really compatible with the new model. How bad is it to drop this in v8, I wonder? 🤔 if we want to keep it we'll have to do some trickery to make this work here.

Reply:
This is very important because we use it to clamp the start timestamp of page load transactions to the browser native measurements for when the request started.

Do you have anything in mind on how we could still have this functionality?

Reply:
IMHO since this is a browser specific thing we'll probably use some internal/private API for this here... I don't think this is possible to do with OTEL native APIs. so probably we'll have to do something like this:

```ts
function clampSpanStartTime(span: Span, startTime: number) {
  const existingStartTime = spanToJSON(span).start_timestamp;
  if (!existingStartTime || existingStartTime > startTime) {
    (span as BrowserInternalSpan).updateStartTime(startTime);
  }
}
```

or something along these lines, if you know what I mean..? Not pretty, but 🤷